stp_core/loop/looper.py

```diff
@@ -212,9 +212,9 @@ def hasProdable(self, prodable: Prodable=None, name: str=None) -> bool:
             # if no let other stuff run
             await asyncio.sleep(0.01, loop=self.loop)
         dur = time.perf_counter() - start
-        if dur >= 0.5:
-            logger.debug("it took {:.3f} seconds to run once nicely".
-                         format(dur), extra={"cli": False})
+        if dur >= 5:
```

Comment:
Currently during the high load it can be quite expected that one looper run is more than 5 secs. It means that during the highload there will be a lot of such lines.